src/Raven.Server/Rachis/RachisConsensus.cs

```diff
@@ -462,6 +464,14 @@ public string ReadNodeTag(TransactionOperationContext context)
             return readResult == null ? InitialTag : readResult.Reader.ToStringValue();
         }
 
+        public string ReadPreviousNodeTag(TransactionOperationContext context)
```

Comment:
Can a server have _multiple_ old tags?

Reply:
You can call `HardResetToNewCluster` multiple times with different tag every time.

Reply:
Wouldn't that put us in the same situation?

Reply:
Shouldn't, because on every tag update we keep also the old one.

Reply:
ok